.github/actions/comment-preview-on-pr/app/main.py

```diff
@@ -0,0 +1,70 @@
+import logging
```

Comment:
Where is this script coming from? Have we used it elsewhere? Why are we not using the existing custom action that we already have in this very repo for commenting?